llvm_util/llvm2alive.cpp

```diff
@@ -270,13 +270,30 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
                                     !known);
     unique_ptr<Instr> ret_val;
     auto argI = fn->arg_begin(), argE = fn->arg_end();
+    auto alist = i.getAttributes();
+    uint64_t argidx = 0;
+
     for (auto &arg : args) {
       ParamAttrs attr;
```

Comment:
can we reuse handleAttribute() here?

Reply:
There are a few encodings that are unsupported at a callsite yet (nonnull/readnone/readonly). I think it might not be okay to show them in the printing?

Reply:
I left them as a TODO comment at the place, with a minor update to the handleAttributes (so the comment makes sense)